### ISSUE TITLE
Cannot achieve result: hide text of attacks Enemy<->Other Player, but show all attacks Enemy<->Player

### ISSUE THREAD

**Issue by Alice-Storey** [Aireil/FlyTextFilter#4]
1. Input: NamedAttack all unchecked
result: 
shows player -> enemy
shows enemy -> player
shows otherPlayer -> enemy

2. Input: NamedAttack, enable only Hide On Others
hides player -> enemy
shows enemy -> player
hides otherPlayer -> enemy

3. Input: NamedAttack, enable only Hide From Others
shows player -> enemy
hides enemy -> player
hides otherPlayer -> enemy

4. (no input found for result)
desired result:
show player->enemy
show enemy->player
hide otherPlayer->enemy

**Comment by faywinne** [comment]
It would make more sense to have settings where either:
1. checkbox options for "enable to SHOW text if it meets criteria" instead of "enable to HIDE text if it meets criteria"
2.  alternatively, existing settings would work if other player was differentiated from other npc

**Comment by Aireil** [comment]
Correct, this is currently impossible to achieve. I'm planning a complete rework of the configuration that will make it possible, but no estimated time.

**Comment by Aireil** [comment]
With 4.0.0.0, you can filter from source Others to target Others to hide those (including Party as well if you don't want those either).

It will be in testing once this PR is merged: https://github.com/goatcorp/DalamudPlugins/pull/2687
